README.md

```diff
@@ -10,6 +10,11 @@ https://www.pythonlodz.org/
 
 
 ## Local install and setup
+Before the next step, be sure the following OS libraries are installed:
```

Comment:
Could you split the local install section into 2 subsections:
- Linux
- Docker

The dev libraries below are for Linux only

Reply:
done